scripts/qiita-load-qebil-downloads

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from time import sleep
+from glob import glob
+from os.path import isdir, basename, join
+from shutil import copyfile
+
+from qiita_db.study import Study
+from qiita_db.artifact import Artifact
+from qiita_db.commands import (
+    load_study_from_cmd, load_sample_template_from_cmd,
+    load_prep_template_from_cmd)
+from qiita_db.util import get_data_types, get_mountpoint
+
+
+SLEEP_TIME = 10
+EBIDIR = '/panfs/panfs1.ucsd.edu/panscratch/qiita/qebil/vertebrates/'
+data_types = set([x.replace(' ', '_') for x in get_data_types()])
+
+for folder in glob(f'{EBIDIR}/*'):
+    warnings = []
+    extra_notes = dict()
+    if not isdir(folder):
+        print(f'Ignoring: {folder}')
+        continue
+    # note necessry but nice for debugging
+    print(f'\n\n\n+++> Processing {folder}, you have {SLEEP_TIME} '
+          'seconds to ctrl-c')
+    sleep(10)
+
+    files = glob(f'{folder}/*')
+    files_used = []
+    qebil_status_fp = [f for f in files if f.endswith('qebil_status')][0]
+    with open(qebil_status_fp, 'r') as fp:
+        qebil_status = fp.readlines()[0]
+    if 'complete' not in qebil_status:
+        print(f'Skipping {qebil_status_fp}, not ready: {qebil_status}')
+        continue
+    files_used.append(qebil_status_fp)
+
+    title_fp = [f for f in files if f.endswith('_study_title.txt')][0]
+    files_used.append(title_fp)
+    config_fp = [f for f in files if f.endswith('_study_config.txt')][0]
+    files_used.append(config_fp)
+    sample_fp = [f for f in files if f.endswith('_sample_info.tsv')][0]
+    files_used.append(sample_fp)
+
+    with open(title_fp, 'r') as fp:
+        title = fp.readlines()[0]
+
+    if Study.exists(title):
+        print(f'======> {folder}: {title} already loaded')
+        continue
+
+    with open(config_fp, 'r') as fp:
+        study = load_study_from_cmd('qiita.help@gmail.com', title, fp)
+
+    study.autoloaded = True
+    study.ebi_study_accession = study.info['study_alias'].split(';')[0]
+    st = study.sample_template
+    sample_info = load_sample_template_from_cmd(sample_fp, study.id)
+    st.ebi_sample_accessions = st.get_category('secondary_sample_accession')
+    st.biosample_accessions = st.get_category('sample_accession')
```

Comment:
Out of curiosity, would it be valuable to test if the accessions were observed already in qiita in addition to the study title? although, I'm not sure what action it would suggest if they were :/